### DISCUSSION TITLE
Push Notifications with FCM not working on iOS

### DISCUSSION THREAD

**Discussion by paulsbin** [WalletConnect/walletconnect-monorepo#5290] [Developer Support]
I followed all the steps according to the documentation on the integration of push notifications for iOS https://docs.walletconnect.com/walletkit/ios/notifications/push
https://docs.walletconnect.com/advanced/push-server

I used WalletConnect Cloud and Google FCM Integration

After calling the method
`try await Web3Wallet.instance.register(deviceToken: deviceToken, enableEncrypted: true)`, 
after a while I get `logger.debug("Successfully registered at Push Server")`

But notifications do not come in the TestFlight build
Where can I track why notifications do not come?

**Comment by llbartekll** [comment]
hey @paulsbin, when you configure Web3Wallet try to set the environment to .sandbox
`Web3Wallet.configure(.., .., .., environment: .sandbox)`

**Comment by paulsbin** [comment]
@llbartekll My current configuration is 
`Web3Wallet.configure(metadata: metadata, crypto: DefaultCryptoProvider(), environment: .sandbox)`
I tried with `.sandbox `and with `.production`, but i dont get result

**Reply by llbartekll** [reply]
@geekbrother can you check whey they are not delivered?

**Reply by paulsbin** [reply]
@geekbrother What information should I provide to check why they are not being delivered?
